demisto_sdk/commands/validate/tests/BA_validators_test.py

```diff
@@ -930,3 +933,113 @@ def test_IsDeprecatedCorrectlyValidator_is_valid():
     for result in results:
         assert result.content_object.deprecated
         assert result.content_object.description == "Some description"
+
+
+@pytest.mark.parametrize(
+    "content_items",
+    [
+        pytest.param(
+            [
+                create_integration_object(
+                    paths=["name"], values=["name_with_space_should_fail "]
+                )
+            ],
+            id="case integration with trailing spaces in name",
+        ),
+        pytest.param(
+            [
+                create_classifier_object(
+                    paths=["name"], values=["name_with_space_should_fail "]
+                )
+            ],
+            id="case classifier with trailing spaces in name",
+        ),
+        pytest.param(
+            [
+                create_dashboard_object(
+                    paths=["name"], values=["name_with_space_should_fail "]
+                )
+            ],
+            id="case dashboard with trailing spaces in name",
+        ),
+        pytest.param(
+            [
+                create_incident_type_object(
+                    paths=["name"], values=["name_with_space_should_fail "]
+                )
+            ],
+            id="case incident type with trailing spaces in name",
+        ),
+        pytest.param(
+            [create_wizard_object({"name": "name_with_space_should_fail "})],
+            id="case wizard with trailing spaces in name",
+        ),
+    ],
+)
+def test_IsContentItemNameContainTrailingSpacesValidator_is_valid(content_items):
+    """Test validate BA113 - Trailing spaces in content item name
+    Given:
+        A list of content items with names that have trailing spaces.
+    When:
+        The IsContentItemNameContainTrailingSpacesValidator's is_valid method is called.
+    Then:
+        The validator should return a list of ValidationResult objects, each with a message indicating that the content item's name should not have trailing spaces.
+    """
+    assert all(
+        [
+            result.message
+            == "Content item name 'name_with_space_should_fail ' should not have trailing spaces. Please remove."
+            for result in IsContentItemNameContainTrailingSpacesValidator().is_valid(
+                content_items
+            )
+        ]
+    )
+
+
+@pytest.mark.parametrize(
```

Comment:
The `paths=["name"], values=["name_with_space_should_fix "]` are the same every time: can we replace it with parametrization of the methods (e.g. `create_integration_object,  create_classifier_object, ...`) and simplify the test code? (at least for most item types, I see `wizard` is different)